api-template-nodejs/test/test.js

```diff
@@ -1,8 +1,8 @@
 const assert = require('chai').assert;
 const index = require('../index');
 
-describe('Sample', function() {
-  it('tests handler', function(done) {
+describe('Sample', function () {

```

Comment:
To be honest i like "function()" rather than "function ()". Even if we go with this then this is not consistent. Look at "describe()" and "it()". Why not do "describe ()" and "it ()". I agree with other changes but don't like "function ()". Also, don't forget we have code quality enabled where there is jshint. did you check if these changes are supported?

Reply:
well not sure if i agree with all of them still. something we can discuss